<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Time Remain</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="digitCount">
         <number>3</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>认输</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="CheeseView" name="graphicsView"/>
    </item>
    <item>
     <widget class="QSlider" name="verticalSlider">
      <property name="minimum">
       <number>20</number>
      </property>
      <property name="maximum">
       <number>180</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>39</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Create">
    <property name="title">
     <string>&amp;Create</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Pieces"/>
   </widget>
   <widget class="QMenu" name="menu_Join">
    <property name="title">
     <string>&amp;Join</string>
    </property>
    <addaction name="action_Join"/>
   </widget>
   <widget class="QMenu" name="menu_Exit">
    <property name="title">
     <string>&amp;Exit</string>
    </property>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Save">
    <property name="title">
     <string>&amp;Save</string>
    </property>
    <addaction name="action_Save"/>
   </widget>
   <addaction name="menu_Create"/>
   <addaction name="menu_Join"/>
   <addaction name="menu_Save"/>
   <addaction name="menu_Exit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Pieces">
   <property name="text">
    <string>&amp;Pieces</string>
   </property>
  </action>
  <action name="action_Join">
   <property name="text">
    <string>&amp;Join</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CheeseView</class>
   <extends>QGraphicsView</extends>
   <header>cheeseview.h</header>
   <slots>
    <slot>setScalePercent(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>verticalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>graphicsView</receiver>
   <slot>setScalePercent(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>931</x>
     <y>427</y>
    </hint>
    <hint type="destinationlabel">
     <x>840</x>
     <y>427</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
